.gitignore

```diff
@@ -52,3 +52,5 @@ buck-out/
 fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
+
+lib
```

Comment:
Well, I like to work and merge on master and to be able to test the bundles on this branch (or other branches) before publish on npm. So what do you think to leave the lib folder on the master branch?

I'm not sure it's a real problem. What do you think?

Reply:
It's not something I do as I don't like putting minified code onto git.

However, I think I've got a good idea for this. If we run the build as part of the precommit, that should rectify this as the latest build will always be in git.

As an added bonus, people can install from git as opposed to npm if they'd like.